lib/digsync_web/live/group_details_live.ex

```diff
@@ -0,0 +1,14 @@
+defmodule DigsyncWeb.GroupDetailsLive do
+  use DigsyncWeb, :live_view
+  alias Digsync.Accounts.Group
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  def fetch_groups() do
```

Comment:
placeholder, need you to run mix format on save. There should be an option in code, I'm 100% sure you can google it and find it instantly, but you'll need to have a format on save hook to avoid this spacing. It helps a lot to know if your code is compiled as well